### DISCUSSION TITLE
Discussion topics for Feb 7, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by cwinters-usds** [CDCgov/prime-reportstream#4192] [Show and tell]
* How to identify security-related Dependabot (or other) PRs? Balancing open-source visibility and potentially exposing vulnerabilities.
    * Example: https://github.com/CDCgov/prime-reportstream/pull/4012
    * Example: https://github.com/CDCgov/prime-reportstream/pull/3679 
    * What's our typical turnaround time for these once identified?

**Comment by carlosfelix2** [comment]
1. @jimduff-usds @carlosfelix2 Report endpoint response proposed changes to match history API and how to handle the migration
